contrib/ovirt/lib/ovirtlago/__init__.py

```diff
@@ -436,23 +437,28 @@ def flush(self):
 
     def _deploy_host(self, host):
         with LogTask('Deploy VM %s' % host.name()):
-            with LogTask('Wait for ssh connectivity'):
-                host.wait_for_ssh()
-
-            for script in host.metadata.get('ovirt-scripts', []):
-                with LogTask('Run script %s' % os.path.basename(script)):
-                    ret, out, err = host.ssh_script(script, show_output=False)
-
-                if ret != 0:
-                    LOGGER.debug('STDOUT:\n%s' % out)
-                    LOGGER.error('STDERR\n%s' % err)
-                    raise RuntimeError(
-                        '%s failed with status %d on %s' % (
+            ovirt_scritps = host.metadata.get('ovirt-scripts', [])
+            if not ovirt_scritps:
```

Comment:
This is reversed, I meant to do this:

    if not ovirt_scripts:
        return  # nothing to do

    with LogTask():....